src/components/graph/Graph.jsx

```diff
@@ -293,7 +298,16 @@ export default class Graph extends React.Component {
                 () => this.props.onClickNode && this.props.onClickNode(clickedNodeId)
             );
         } else {
-            this.props.onClickNode && this.props.onClickNode(clickedNodeId);
+            if (!this.timeoutID) {
+                this.timeoutID = setTimeout(() => {
+                    this.props.onClickNode && this.props.onClickNode(clickedNodeId);
```

Comment:
Is there any specific reason for this call to be wrapped inside a `setTimeout`?

Reply:
Yes, it represents the time when the Node is waiting to see if another click is coming, and then decides if it should apply the onClickNode or onDoubleClickNode function

Reply:
For the dblclick event, it didn't work, JS and React as a whole have huge problems managing click and doubleclick on the same element and StackOverflow is full of questions regarding this issue.
I have run some tests to try to move the logic to the `Node` component but it doesn't work anymore, nothing is triggered when I double click, I don't understand why ...